### PR TITLE
@broskoski => Fix professional buyer asset package

### PR DESCRIPTION
**Pull request by mzikherman**
Fixes https://github.com/artsy/collector-experience/issues/216

### CODE REVIEW

File: desktop/apps/pro_buyer/pages/landing/client/view.coffee [hunk 1]
```diff
@@ -7,7 +7,7 @@ AuthModalView = require '../../../../../components/auth_modal/view.coffee'
 analyticsHooks = require '../../../../../lib/analytics_hooks.coffee'
 template = -> require('../templates/page.jade') arguments...
 
-module.exports = class ProfessionalBuyerIndexView extends View
+module.exports = class ProfessionalBuyerLandingView extends View
```
Comment (mzikherman) [inline desktop/apps/pro_buyer/pages/landing/client/view.coffee]: Doesn't matter but just figured to be consistent

File: desktop/assets/misc.coffee [hunk 2]
```diff
@@ -16,7 +16,7 @@ routes =
 
   '/personalize': require('../apps/personalize/client/index.coffee').init
 
-  '/professional-buyer': -> require('../apps/pro_buyer/client/index.coffee')
+  '/professional-buyer': require('../apps/pro_buyer/client/index.coffee')
```
Comment (mzikherman) [inline desktop/assets/misc.coffee]: This is the actual fix 😐

Comment (broskoski) [inline desktop/assets/misc.coffee]: 👍

**Comment by gardnerae** [comment]
Cross posting here: 

So is this fix implemented? it doesn't seem to work for me @mzikherman @broskoski (cc: @stasya) 

Apologies if i am asking this prematurely...!

**Pull request merged**